OsmAnd/res/values/strings.xml

```diff
@@ -77,8 +77,8 @@
     <string name="paste_Osmand_data_folder_path">Paste path to the folder with OsmAnd data</string>
     <string name="change_osmand_data_folder_question">Change OsmAnd data folder?</string>
     <string name="move_maps_to_new_destination">Move to the new destination</string>
-    <string name="internal_app_storage_description">Internal storage, hidden from user and other apps, exclusively accessible to OsmAnd</string>
-    <string name="change_data_storage_folder">Change data storage folder</string>
+    <string name="internal_app_storage_description">Internal storage for OsmAnd, (hidden from users and other apps).</string>
```

Comment:
Is it always on internal memory, or could it be "Internal OsmAnd storage"?

Reply:
memory, I think